laa_court_data_api_app/internal/court_data_adaptor_client.py

```diff
@@ -22,13 +22,23 @@ async def get(self, endpoint: str, params: Optional[dict[str, str]] = None,
     async def post(self, endpoint: str, params: Optional[dict[str, str]] = None,
                    headers: Optional[dict[str, any]] = None,
                    body: Optional[any] = None):
+        if headers is None:
+            headers = {}
```

Comment:
Can this conditional be replaced by the following in the params on line 23?
`headers: dict[str, any] | None = {}`

Reply:
This is not what we should be doing as the argument is mutable. More reading around this here https://docs.python-guide.org/writing/gotchas/